pkg/server/options/options.go

```diff
@@ -361,14 +361,15 @@ func mkdirRoot(dir string) error {
 	if dir == "" {
 		return errors.New("missing root directory configuration")
 	}
+	logger := klog.Background().WithValues("dir", dir)
```

Comment:
If we wanted to, we could wire in a context from main.go down to here

Reply:
Didn't seem like it would really benefit us at all, so I didn't